services/ui-src/src/verbiage/pages/mcpar/mcpar-intro.ts

```diff
@@ -1,6 +1,73 @@
+import NavigationSectionsImage from "../../../assets/images/NavigationSections_2x.png";
+import NavigationSectionsSubmissionImage from "../../../assets/images/NavigationSectionsSubmission_2x.png";
+import greenSpreadsheetIcon from "../../../assets/icons/icon_spreadsheet_green.png";
+
 export default {
   intro: {
-    header: "MCPAR Intro",
-    body: "Placeholder page",
+    header: "Before you begin the MCPAR online form",
+  },
+  body: {
+    sections: [
+      {
+        header: "Start by adding all Medicaid managed care programs",
+        body: "You will need to submit one MCPAR for every managed care program in your state. For this MCPAR report, a managed care program is defined by distinct benefits and eligibility criteria articulated in a contract between states and the state’s managed care plans.",
+        widgets: [
+          {
+            type: "iconlessWidget",
+            content: {
+              leftBarColor: "#0071BC",
+              title: "Examples of managed care programs:",
+              descriptionList: [
+                "Health and Recovery Plans (Comprehensive MCO)",
+                "Dental Managed Care",
+              ],
+            },
+          },
+        ],
+      },
+      {
+        header: "Enter data for each program into the online form",
+        body: "The online form sections are organized by state-level, program-level, plan-level indicators. In general, the MCPAR online form matches the organization of the MCPAR Excel workbook. There are some exceptions.",
+        widgets: [
+          {
+            type: "imageWidget",
+            content: {
+              src: NavigationSectionsImage,
+              alt: "Image of side navigation in the application ",
+              additionalInfo: "Preview of the online MCPAR form navigation.",
+            },
+          },
+          {
+            type: "iconWidget",
+            content: {
+              leftBarColor: "#2E8540",
+              icon: greenSpreadsheetIcon,
+              iconDescription: "Excel Workbook Icon",
+              title: "Find in the Excel Workbook",
+              descriptionList: ["A_Program_Info"],
+              additionalInfo:
+                "Use these guides to understand which sections match specific tabs in the Excel workbook.",
+            },
```

Comment:
I'm conflicted about putting colors here since we're trying to stick to theming so much (I know this value probably would need to be added to the theme anyway though). 

I wonder if it would make sense to take the common things from here that all widgets of a certain type could share, for instance, we will be using this spreadsheet widget in like a million places with just slightly different wording. So we could have the type be 'spreadsheetWidget' or something and when it's that type of widget, it gets the spreadsheet icon and green color bar. Just trying to think for the future where we won't be able to pass in icon or what to manage colors in the form.json where the spreadsheet icons will be used a lot. Especially since having all those details here in json puts the burden of getting it right on the json writer in 3 areas (type, color, icon) instead of just one: which type of widget do you want.

Reply:
Agreeing with Brax here in regards to the theming. I think it makes sense to just have the form include something like `leftBarColor: 'green'` and have those mapped against colors from known themes internally. It's a lot less of a mental burden when spinning up a new section than having to go back and find the code for green you used elsewhere.